spec/fare_transactions.schema.json

```diff
@@ -0,0 +1,220 @@
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "amount",
+      "type": "number",
+      "description": "Value of the transaction.",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "currency_type",
+      "type": "string",
+      "rdfType": "https://schema.org/currency",
+      "description": "Currency used for the transaction. References GTFS",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "fare_action",
+      "type": "string",
+      "description": "Indicates the type of action performed.",
+      "constraints": {
+        "required": true,
+        "enum": [
+          "Unknown action type",
+          "Purchase",
+          "Enter",
+          "Exit",
+          "Transfer entrance",
+          "Transfer exit",
+          "Add",
+          "Activate",
+          "Adjust",
+          "Other"
+        ]
+      }
+    },
+    {
+      "name": "trip_id_performed",
+      "type": "string",
+      "title": "ID referencing trips_performed.trip_id_performed",
+      "description": "Identifies the trip performed. May be null if the fare collection device is NOT located on a vehicle. May be null if on a vehicle but trip-level data is unavailable, in which case the data would be associated with the vehicle."
+    },
+    {
+      "name": "stop_sequence",
+      "type": "integer",
+      "title": "positive integer referencing stop_times.stop_sequence",
+      "description": "Ordered stop the vehicle is serving on a particular trip. May be null if the fare collection device is NOT located on a vehicle. May be null if on a vehicle but stop-level data is unavailable, in which case the data would be associated with the vehicle and/or trip. References GTFS",
+      "constraints": {
+        "minimum": 0
+      }
+    },
+    {
+      "name": "vehicle_id",
+      "type": "string",
+      "title": "ID referencing vehicles.vehicle_id",
+      "description": "Identifies the vehicle. May be null if collection device is NOT located on a vehicle. May be null if on a vehicle but vehicle data is unavailable, in which case the data would be associated with a trip and/or stop."
+    },
+    {
+      "name": "device_id",
+      "type": "string",
+      "title": "ID referencing devices.device_id",
+      "description": "Identifies the ITS device on which the fare transaction was performed. May be null if only a single device is reporting fare transactions on a vehicle and vehicle_id is provided. May be null if only a single device is reporting fare transactions at a stop and stop_id is provided."
+    },
+    {
+      "name": "fare_id",
+      "type": "string",
+      "title": "ID referencing fare_attributes.fare_id",
+      "description": "Identifies a fare class, as included in the GTFS Fare_attributes file. References GTFS"
+    },
+    {
+      "name": "stop_id",
+      "type": "string",
+      "title": "ID referencing stops.stop_id",
```

Comment:
Here the "references GTFS" is in the description, but the GTFS table and field is in the title. I think if we provide a table and field we should specify the source if it's not this TIDES spec